src/core/Akka.Remote.Tests.MultiNode/NewRemoteActorSpec.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System;
+using Akka.Actor;
+using Akka.Configuration;
+using Akka.Remote.TestKit;
+using Akka.TestKit;
+using Xunit;
+
+namespace Akka.Remote.Tests.MultiNode
+{
+    public class NewRemoteActorMultiNodeSpecConfig : MultiNodeConfig
+    {
+        #region Internal actor classes
+
+        public class SomeActor : UntypedActor
+        {
+            protected override void OnReceive(object message)
+            {
+                if(message.Equals("identify"))
+                    Sender.Tell(Self);
+            }
+        }
+
+        public class SomeActorWithParam : UntypedActor
+        {
+            private string _ignored;
+
+            public SomeActorWithParam(string ignored)
+            {
+                _ignored = ignored;
+            }
+
+
+            protected override void OnReceive(object message)
+            {
+                if (message.Equals("identify"))
+                    Sender.Tell(Self);
+            }
+        }
+
+        #endregion
+
+        readonly RoleName _master;
+        public RoleName Master { get { return _master; } }
+        readonly RoleName _slave;
+        public RoleName Slave { get { return _slave; } }
+
+        public NewRemoteActorMultiNodeSpecConfig()
+        {
+            _master = Role("master");
+            _slave = Role("slave");
+
+            CommonConfig =
+                DebugConfig(false)
+                    .WithFallback(ConfigurationFactory.ParseString("akka.remote.log-remote-lifecycle-events = off"));
+
+            DeployOn(Master, @"
+                /service-hello.remote = ""@slave@""
+                /service-hello-null.remote = ""@slave@""
+                /service-hello3.remote = ""@slave@""
+            ");
+
+            DeployOnAll(@"/service-hello2.remote = ""@slave@""");
+        }
+    }
+
+    public class NewRemoteActorSpecNode1 : NewRemoteActorSpec { }
+    public class NewRemoteActorSpecNode2 : NewRemoteActorSpec { }
+
+    public class NewRemoteActorSpec : MultiNodeSpec
+    {
+        private NewRemoteActorMultiNodeSpecConfig _config;
+
+        public NewRemoteActorSpec()
+            : this(new NewRemoteActorMultiNodeSpecConfig())
+        {
+        }
+
+        public NewRemoteActorSpec(NewRemoteActorMultiNodeSpecConfig config) : base(config)
+        {
+            _config = config;
+        }
+
+        protected override int InitialParticipantsValueFactory
```

Comment:
@Aaronontheweb Is there any specific reason the name of this property is not just `InitialParticipants` as in scala version?